check_times.py

```diff
@@ -78,17 +79,38 @@ def calc_time(lines: list[str]) -> None:
 
 
 if __name__ == "__main__":
-    print("Build times (HTML only; English)")
-    print("=======================")
-    print()
-    calc_time(get_lines("docsbuild-only-html-en.log"))
-
-    print("Build times (HTML only)")
-    print("=======================")
-    print()
-    calc_time(get_lines("docsbuild-only-html.log"))
-
-    print("Build times (no HTML)")
-    print("=====================")
-    print()
-    calc_time(get_lines("docsbuild-no-html.log"))
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
```

Comment:
We could mirror the `build_docs.py` args here? 

```
    parser.add_argument(
        "--select-output",
        choices=("no-html", "only-html", "only-html-en"),
        help="Choose what outputs to build.",
    )
```

Reply:
Something like this? https://github.com/python/docsbuild-scripts/pull/307/commits/45f56e5e9785488982a9531b2d7bd7f4d4282e99